similarity-search/src/main/java/org/phenotips/similarity/internal/SolrSimilarPatientsFinder.java

```diff
@@ -204,4 +214,36 @@ private long count(SolrQuery query)
         }
         return 0;
     }
+
+    private Collection<String> getPresentPhenotypeTerms(Patient patient)
+    {
+        Collection<String> termIds = new ArrayList<String>();
```

Comment:
Just curious if we have a good metric to measure the quality of matches? From what Orion was saying today @ standup, doctors seem to prefer not the matches we theoretically think are good?